src/transformers/modeling_utils.py

```diff
@@ -5083,6 +5083,25 @@ def loss_function(self):
             loss_type = "ForCausalLM"
         return LOSS_MAPPING[loss_type]
 
+    @property
+    def compiled_call(self):
```

Comment:
This could also use [`functools.cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property) but it's also fine as is.